packages/python-google-compute-engine/google_compute_engine/instance_setup/instance_setup.py

```diff
@@ -122,6 +133,8 @@ def _GenerateSshKey(self, key_type, key_dest):
     Args:
       key_type: string, the type of the SSH key.
       key_dest: string, a file location to store the SSH key.
+    Returns:
```

Comment:
Please add a new line between the args and return sections.

Reply:
Done.